### PR TITLE
Clarify recommendations for installation options

### PR DESCRIPTION
**Pull request by victorlin**
_[preview](https://nextstrain--106.org.readthedocs.build/en/106/install.html)_

### Description of proposed changes

Previously, the text:

1. Recommended Docker without giving any explanation
2. Did not direct Windows users how to choose between Windows and WSL on Windows

This change:

1. Adds a note on how to choose between Docker vs. Native and Windows vs. WSL on Windows
2. Removes the universal recommendation for Docker

### Related issue(s)

_N/A_

### Testing

_N/A_

### CODE REVIEW

File: src/install.rst
Comment (tsibley) [inline src/install.rst:46]: Hmm. I think I know what you intend here, but as it's worded, this reads as potentially misleading to me. I think, but correct me if wrong, that the intent is to communicate that if you want to use other arbitrary bioinformatics tools _within a `nextstrain build` run_ then it'll be (much) easier to do that with "native" installation via Conda. That emphasized bit of scoping seems important to me, as there's nothing about either installation method that makes it harder or easier to use other bioinformatics tools on Nextstrain inputs before hand, outputs afterwards, etc.

It might also be worth mentioning in this note that _both_ options can be followed, if you know up-front you want the flexibility to switch between then or (more realistically perhaps) you realize later you initially choose the wrong option for your needs.

Relatedly, ISTM that we're going to say "use Conda if you want to use other bioinformatics tools in your `nextstrain build` run", then we should also probably include some information about how to actually do that (i.e. how to install other arbitrary things into the Conda env). This implies to me that a separate doc page might be in order, something like a how-to _Use Nextstrain with other bioinformatics tools_.

File: src/install.rst
Comment (j23414) [inline src/install.rst:51]: I like the explanation on "when to choose WSL", based on my past experience helping researchers to get set up on a UNIX-like environment to use the HPC, and learn various bioinformatic toolsets.

**Review by j23414 — APPROVED** [review]

File: src/install.rst
Comment (huddlej) [reply src/install.rst:46]: I agree with @tsibley about the potential for confusion. Some of this choice comes down to whether users want to use Docker/Conda or not. We've heard from folks at office hours who do not want to use Docker (or Conda), and they will choose the installation platform that they do prefer. For people who don't know what they prefer, it still makes sense to me to recommend Docker (and flag Docker as the recommended approach).

Other components of this choice that seem most relevant are well-described here:

  - "Docker provides the most predictable environment"
  -  managing your own conda environments

In the past we've talked about this kind of decision as "if you know what you're doing, here are your options". The same context applies to the Windows vs. Windows/WSL choice.

We should discuss this more as a small group and refine this PR based on our discussion...

Comment (huddlej) [reply src/install.rst:46]: I agree with @tsibley about the potential for confusion. Some of this choice comes down to whether users want to use Docker/Conda or not. We've heard from folks at office hours who do not want to use Docker (or Conda), and they will choose the installation platform that they do prefer. For people who don't know what they prefer, it still makes sense to me to recommend Docker (and flag Docker as the recommended approach).

Other components of this choice that seem most relevant are well-described here:

  - "Docker provides the most predictable environment"
  -  managing your own conda environments

In the past we've talked about this kind of decision as "if you know what you're doing, here are your options". The same context applies to the Windows vs. Windows/WSL choice.

We should discuss this more as a small group and refine this PR based on our discussion...

Comment (tsibley) [reply src/install.rst:46]: > We should discuss this more as a small group and refine this PR based on our discussion...

+1

**Comment by tsibley** [comment]
Bumping this.

**Comment by j23414** [comment]
Is there anything blocking this from being merged? The text looks correct to me.

**Comment by tsibley** [comment]
@j23414 There are unaddressed review comments above about what information to convey and how to convey it. I bumped this PR earlier only to bring attention to it again, not to suggest merge as-is. I do still think we want to have a small focused discussion on the topics brought up in the comments.

**Comment by victorlin** [comment]
From [today's meeting](https://docs.google.com/document/d/1dODYr-3T8LAQt-ewMZKNrcy1Wvy70Fw-h8exXbIwbyM/), we plan to incorporate these recommendations into #123 and refrain from making any large changes to the installation docs outside of that PR, such as reordering the tab hierarchy. It is simplest to keep the existing recommendations until that PR is merged.

File: src/install.rst
Comment (tsibley) [reply src/install.rst:51]: I'm using language like the above in https://github.com/nextstrain/docs.nextstrain.org/pull/123.

**Pull request closed without merging**
